packages/editor-sample/src/documents/blocks/EmailLayout/EmailLayoutEditor.tsx

```diff
@@ -36,68 +36,78 @@ export default function EmailLayoutEditor(props: EmailLayoutProps) {
   const currentBlockId = useCurrentBlockId();
 
   return (
-    <div
-      onClick={() => {
-        setSelectedBlockId(null);
-      }}
-      style={{
-        backgroundColor: props.backdropColor ?? '#F5F5F5',
-        color: props.textColor ?? '#262626',
-        fontFamily: getFontFamily(props.fontFamily),
-        fontSize: '16px',
-        fontWeight: '400',
-        letterSpacing: '0.15008px',
-        lineHeight: '1.5',
-        margin: '0',
-        padding: '32px 0',
-        width: '100%',
-        minHeight: '100%',
-      }}
-    >
-      <table
-        align="center"
-        width="100%"
+    <Fragment>
+      {props.previewText && (
+        <div
+          style={{ display: 'none', maxHeight: '0px', overflow: 'hidden', 'mso-hide': 'all' } as React.CSSProperties}
```

Comment:
[disclaimer: not a maintainer]

I don't think you don't need to add this `mso-hide` to the Editor as it won't be "visible", it's for Outlook only and it's throwing this:

![image](https://github.com/user-attachments/assets/b8210ea3-060b-4ae7-b7ff-5d07fa45f9b9)